src/System.Net.WebSockets/src/System/Net/WebSockets/WebSocketReceiveResult.cs

```diff
@@ -31,7 +31,7 @@ public WebSocketReceiveResult(int count, WebSocketMessageType messageType, bool
             CloseStatusDescription = closeStatusDescription;
         }
 
-        public int Count { get; }
+        public int Count { get; set;}
```

Comment:
Is this changing a public API? It also doesn't seem like this is something that should be publicly set, based off of the documentation.